i18n/zh.yml

```diff
@@ -274,7 +274,6 @@ components:
     incoming_at: 接近 {stop}
     stopped_at: 巴士车门在 {stop} 打开
     travelingAt: 以 {milesPerHour}
-  vehicleName: '车辆 {vehicleNumber}'
```

Comment:
interesting ... this should not be deleted, rather indented

Reply:
Correct. See updated change in line below (I reverted the delete in Weblate).

Reply:
Thank you!